similar/main/gamecntl.cpp

```diff
@@ -1937,6 +1937,8 @@ window_event_result ReadControls(const d_event &event)
 
 	if (event.type == EVENT_KEY_COMMAND)
 	{
+		window_event_result result;
```

Comment:
OK, this is a habit I had since coding for good old Mac OS 9 (which @zicodxx knows all about - endless fixes to accommodate ;) ), which doesn't like variable declarations after lines of code. Since I will no longer be supporting Mac OS 9 (or PowerPC Macs for that matter), I will do it that way.